src/mlpack/methods/ann/loss_functions/binary_cross_entropy_loss.hpp

```diff
@@ -77,9 +81,9 @@ class BCELoss
   //! Modify the epsilon.
   double& Eps() { return eps; }
 
-  //! Get the reduction.
+  //! Get the type of reduction used.
```

Comment:
Perhaps we should use `Get the reduction type, represented as boolean (false 'mean' reduction, true 'sum' reduction).`?